tox.ini

```diff
@@ -28,6 +30,15 @@ commands=
     chmod 755 {toxinidir}/bin/minio
     screen -dmS {envname} {toxinidir}/bin/minio server {toxinidir}{env:MINIO_DATA_DIR}
 
+[testenv:prometheus]
+changedir={toxinidir}/bin/prometheus
+commands=
+    wget https://github.com/prometheus/prometheus/releases/download/v2.18.1/prometheus-2.18.1.linux-amd64.tar.gz -O temp.tar.gz
+    tar --strip-components=1  -zxvf temp.tar.gz
+    rm temp.tar.gz
+    chmod 755 prometheus
+    screen -dmS {envname} ./prometheus --config.file=prometheus.yml
```

Comment:
Is the prometheus config file part of the download?

Reply:
Yes it is.